lib/simulator-xcode-6.js

```diff
@@ -457,6 +462,12 @@ class SimulatorXcode6 extends EventEmitter {
       args.push('-ConnectHardwareKeyboard', opts.connectHardwareKeyboard ? '1' : '0');
     }
 
+    if (_.isBoolean(opts.disablePasteboardAutomaticSync)) {
+      // Improve launching simulator performance
+      // https://github.com/WebKit/webkit/blob/master/Tools/Scripts/webkitpy/xcode/simulated_device.py#L413
+      args.push('-PasteboardAutomaticSync', '0');
```

Comment:
what is the minimum SDK version where this arg is supported?

Reply:
It looks Xcode 9.3 simulator has it, at least
![image](https://user-images.githubusercontent.com/5511591/74390359-5d671200-4e44-11ea-9c18-5dea1b29b97a.png)

Reply:
I was just thinking about this option is available since simulator9 and we have the code inside simulator6, but I would say additional args does not affect older simulators functionality anyway.